frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -109,6 +110,14 @@ export const notebookLogic = kea<notebookLogicType>([
                 clearLocalContent: () => null,
             },
         ],
+        localTextContent: [
```

Comment:
I was trying to avoid writing a parser for the JSON content... 🤔

Reply:
Sorry, I don't think I was super clear here. I more meant why don't we just call `editor.getText()` when the notebook is being saved. We only store `localContent` because that is what is rendered locally & protects against data loss in case the autosave does not work or you go offline. Given we don't reference `localTextContent` anywhere in the UI maybe we could not recompute this alongside `localContent`